packages/angular/projects/clr-angular/src/utils/focus-trap/focus-trap.directive.ts

```diff
@@ -61,22 +64,19 @@ export class FocusTrapDirective implements AfterViewInit, OnDestroy {
       return;
     }
     const nativeElement: HTMLElement = this.el.nativeElement;
-    if (this._config.strict && event.target && !nativeElement.contains(event.target)) {
-      // When the focus trap is global, always steal focus back if it goes outside
-      nativeElement.focus();
-    } else if (event.target === this.bottomReboundEl && nativeElement.contains(this.previousTrappedActiveElement)) {
-      // When the focus trap is local, if the user navigates via keyboard to the end element from within the trap, move to top
-      nativeElement.focus();
-    } else if (event.target === this.topReboundEl) {
-      // When the focus trap is local, if the user navigates via keyboard back to start element from within the trap, move to bottom
-      // @TODO implement an acceptable solution to SHIFT+TAB navigation
-    } else if (event.target !== nativeElement && !nativeElement.contains(event.target)) {
-      // If a user has escaped the trap using the mouse
-      // relax, don't do it, when you want to go to it, living those dreams, scheme those schemes, hit me with those laser beams
-      this.localFocusEscaped = true;
+    if (!nativeElement.contains(event.target)) {
```

Comment:
Why not add `nativeElement.contains(event.target)` to the early return on ln 63?

</nit>

Reply:
We sure can... Will update. Thank you!